### PR TITLE
in-place update of DB dependency to 18.1.40 in Perl 5.30.2 easyconfig (because sources for DB 18.1.32 are no longer available)

### PR DESCRIPTION
**Pull request by dithwick**
(created using `eb --new-pr`) Depends on #17318

### CODE REVIEW

File: easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -21,7 +21,7 @@ dependencies = [
     ('expat', '2.2.9'),  # for XML::Parser
     ('ncurses', '6.2'),  # for Term::ReadLine::Gnu
     ('libreadline', '8.0'),  # for Term::ReadLine::Gnu
-    ('DB', '18.1.32'),  # for DB_File
+    ('DB', '18.1.40'),  # for DB_File
```
Comment (boegel) [inline easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb:24]: in-place updates like this are pretty painful...

Another solution to the source tarball for DB 18.1.32 no longer being available is to upload it at https://sources.easybuild.io, which is automatically considered as a fallback URL by EasyBuild.
That's only possible if the software license allows for redistribution though...

Comment (dithwick) [reply easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb:24]: I don't have a copy of that source file myself. Maybe someone else does (as you say, depending on the license).

Comment (boegel) [reply easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb:24]: DB 18.1.x is released under AGPL (+ UPL for examples) license, so sources can be redistributed -> see https://sources.easybuild.io/d/DB/

@dithwick With that, I think we can close this PR as well as #17318?

Comment (dithwick) [reply easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb:24]: Yep, that's fine with me of course

**Comment by jfgrimm** [comment]
closed as per: https://github.com/easybuilders/easybuild-easyconfigs/pull/17319#discussion_r1108184277

**Pull request closed without merging**